style.css

```diff
@@ -0,0 +1,111 @@
+main {
```

Comment:
px are a fixed measurement where as ems and rems are not fixed. I think you can have a mix of them, since you want to have certain things fixed and others fluid. I did change everything to ems so there is not a mix of ems and rems